collections/business/general-notifications/event-general-notifications/batch-status-updated/event-general-notifications.json

```diff
@@ -23,12 +23,37 @@
           },
           "engagementTemplates": [
             {
-              "path": "/batch-status-updated/in-app-notification/en_US/engagement-template.json",
+              "path": "/batch-status-updated/in-app-notification/en_US/batch-engagement-template.json",
               "templateName": "batch-status-updated-in-app-notification",
               "locale": "en_US"
             }
           ]
         },
+        {
+          "id": "direct-debit-status-updated",
+          "name": "Direct Debit Status Updated",
+          "userPreferenceEnabled": true,
+          "additionalFilterLogic": "(#triggeredEvent['status'] == 'REJECTED' || (#triggeredEvent['status'] == 'PROCESSED' && #triggeredEvent['rejectedTransactionsCount'] >= 1)) && #triggeredEvent['type'] == 'SEPADD'",
+          "deliveryOptions": [
+            {
+              "channel": "in-app-notification",
+              "channelSettings": "/batch-status-updated/in-app-notification/in-app_channel-settings.json",
+              "isEnabledByDefault": true,
+              "isManageableByUser": false,
+              "routeInfo": {
+                "whereTo": "direct-debit-view",
+                "data": "{\"id\":\"{{event.id}}\",\"batchType\":\"{{event.type}}\"}"
+              },
+              "engagementTemplates": [
+                {
+                  "path": "/batch-status-updated/in-app-notification/en_US/direct-debit-engagement-template.json",
+                  "templateName": "batch-status-updated-in-app-notification",
+                  "locale": "en_US"
+                }
+              ]
+            }
+          ]
+        },
```

Comment:
same here, could it be added in wrong section in the file?

Reply:
Solved as above